cluster/gce/config-test.sh

```diff
@@ -132,3 +132,5 @@ OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+# Overlay network settings
+OVERLAY_NETWORK=${OVERLAY_NETWORK:-true}
```

Comment:
Yeah that's what I'm doing with the coming pr. I'm attaching to network provider, but translating to a kubelet flag called --experimental-flannel-overlay to differentiate it from the other network providers that aren't tightly integrated yet. Re: default to flannel, so kube-up produces a flannel cluster?